core/src/main/java/com/rim/IceField/Game.java

```diff
@@ -117,4 +118,184 @@ public static void Turn(PlayerBase player) throws Exception {
             }
         }
     }
+
+    //Will be continuously called in the game loop.
+    public void UserInteraction(ArrayList<String> text, PlayerBase player)
+    {
+        if(text.get(0).equals("move ")) {
+            if (text.get(1).equals("NORTH")) {
+
+                try {
+                    player.move("north", this.getMap());
+                    System.out.println("Action accepted!");
+                } catch (Exception e) {
+                    e.printStackTrace();
+                }
+            } else if (text.get(1).equals("SOUTH")) {
+                try {
+                    player.move("south", this.getMap());
+                    System.out.println("Action accepted!");
+                } catch (Exception e) {
+                    e.printStackTrace();
+                }
+            } else if (text.get(0).equals("LEFT")) {
+                try {
+                    player.move("west", this.getMap());
+                    System.out.println("Action accepted!");
+                } catch (Exception e) {
+                    e.printStackTrace();
+                }
+            } else if (text.get(0).equals("RIGHT")) {
+                try {
+                    player.move("east", this.getMap());
+                    System.out.println("Action accepted!");
+                } catch (Exception e) {
+                    e.printStackTrace();
+                }
+            }
+        }
+
+        else if(text.get(0).equals("use")) {
```

Comment:
here can be applied the same principle as above, it would make the code a little bit more clean